spec/rubocop/cop/rspec/capybara/negation_matcher_spec.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::Capybara::NegationMatcher, :config do
+  let(:cop_config) { { 'EnforcedStyle' => enforced_style } }
+
+  context 'with EnforcedStyle `have_no`' do
+    let(:enforced_style) { 'have_no' }
+
+    %i[selector css xpath text title current_path link button
+       field checked_field unchecked_field select table
+       sibling ancestor].each do |matcher|
+      it 'registers an offense when using ' \
+        '`expect(...).not_to have_#{matcher}`' do
+        expect_offense(<<~RUBY, matcher: matcher)
+          expect(page).not_to have_#{matcher}
+                       ^^^^^^^^^^^^^{matcher} Use `expect(...).not_to have_#{matcher}` instead of `expect(...).to have_no_#{matcher}`.
```

Comment:
It seems there's duplication here. `^^^^^^` already highlight what we consider bad.

Reply:
Here, it was `Use bad instead of good` when it should have been `Use good instead of bad` to begin with.